apps/code-examples/src/app/code-examples/pages/page/record-page-tabs-layout-demo/record-page-overview-tab.component.ts

```diff
@@ -99,3 +100,11 @@ export class RecordPageOverviewTabComponent {
     },
   ];
 }
+
+export class RecordPageOverviewTabHarness extends ComponentHarness {
```

Comment:
Heads up that up until now we do not show consumers how to use `queryHarness` for their own component harnesses. This is the first time in code examples we are demoing that. This would set a potential standard in code examples and I'd like to chat about whether this is ok or if we should do something else.